SUSEConnect.example

```diff
@@ -15,3 +15,6 @@
 
 ## Automatically agree to extension and module license confirmation prompts (default: false)
 # auto_agree_with_licenses: false
+
+## Enable system uptime tracking. The system uptime log will be sent to SCC/RMT as part of heartbeat.
```

Comment:
Instead of `heartbeat`, call it `keepalive`, since this is how we call it in the documentation as well as in the flag itself.

Reply:
Done